charts/irs-helm/README.md

```diff
@@ -1,16 +1,18 @@
+[[Back to main README](../../README.md)]
+
 # Item Relationship Service Helm Chart
 
 This chart installs the Item Relationship Service and its direct dependencies.
 This includes:
 
- - Minio Object Storage (enabled by default)
- - Prometheus (disabled by default) 
- - Grafana (disabled by default) 
+ - [Minio Object Storage](https://min.io) (enabled by default)
+ - [Prometheus](https://prometheus.io) (disabled by default) 
+ - [Grafana](https://grafana.com/grafana) (disabled by default) 
 
 ## Prerequisites
 
-- Kubernetes 1.19+
-- Helm 3.2.0+
+- [Kubernetes](https://kubernetes.io) 1.19+
```

Comment:
Please remove version information here  - therefor the compatibility matrix and the project is used. This might be outdated soon.

Reply:
Hi @mkanal, the versions were already there. I just added a link to the websites. Furthermore, currently the COMPATIBILITY_MATRIX.md does not include helm and kubernetes.

Reply:
So my recommendation would be to add this to the COMPATIBILITY_MATRIX.md and delte it here

Reply:
Prerequisites in the chart README are required by TRG 5.02. Do not remove it from here. 
I'm not sure if we should add this to the compatibility matrix since that is a requirement for the IRS Helm Chart and not for the IRS application in general